Makefile

```diff
@@ -335,26 +335,63 @@ shadow-server: export TARGET := clojure
 shadow-server:##@ Start shadow-cljs in server mode for watching
 	yarn shadow-cljs server
 
+_clojure-test:
+	yarn install && \
+	yarn shadow-cljs compile mocks && \
+	yarn shadow-cljs compile test && \
+	node --require ./test-resources/override.js "$$SHADOW_OUTPUT_TO"
+
+_clojure-test-watch:
+	yarn install && \
+	yarn shadow-cljs compile mocks && \
+	nodemon --exec "yarn shadow-cljs compile test && node --require ./test-resources/override.js $$SHADOW_OUTPUT_TO" -e cljs
+
+test: export TARGET := clojure
```

Comment:
What if you move this to subtasks too?
Will it work?

Reply:
Good idea and it does work locally. Pushed now, hopefully it works in the CI too.